src/Time.mo

```diff
@@ -13,3 +13,20 @@ module {
     public let now : () -> Int =
       func () : Int = Prim.nat64ToNat(Prim.time());
 }
+
+// The following example illustrates using the system time:
```

Comment:
If you want this to show up in `mo-doc` generated documentation you need to move it into the module header or into the comment on `now`. When you move it into the header make sure to include a new line to denote a paragraph like so:

```
/// System time
///
/// The following example illustrates using the system time:
/// ....
```

(you also need to start this line with a triple `/`)